packages/shared/src/lib/abstractBoard/helper/SierpinskyBoard.ts

```diff
@@ -2,15 +2,15 @@ import { Vector2D } from "./types/Vector2D";
 import { Intersection } from "./types/intersection";
 
 export function createSierpinskyBoard(depth: number): Intersection[] {
-  const sideLength = Math.pow(2, depth + 1);
```

Comment:
Looks much better!

Just to make sure I understand the name change... is this the radius in question?

<img width="817" alt="Screenshot 2025-01-02 at 9 01 48 PM" src="https://github.com/user-attachments/assets/55bcc43b-291e-49cd-9d41-900020bc196a" />

Reply:
> Just to make sure I understand the name change... is this the radius in question?

Yes exactly 🙂